src/tls/Makefile.common

```diff
@@ -1,10 +1,33 @@
-HACL_HOME?=../../../hacl-star
-FSTAR_HOME?=../../../FStar
-MITLS_HOME?=../..
-KREMLIN_HOME	?= ../../../kremlin
-MLCRYPTO_HOME	?= ../../../MLCrypto
+HACL_HOME       ?= ../../../hacl-star
+FSTAR_HOME      ?= ../../../FStar
+MITLS_HOME      ?= ../..
+KREMLIN_HOME    ?= ../../../kremlin
+MLCRYPTO_HOME   ?= ../../../MLCrypto
+FLAVOR          ?= Model
+CONCRETE_FLAVOR ?=
 
-INCLUDE_PATHS= \
+SHELL=/bin/sh
```

Comment:
this is good to enforce uniformity, have you noticed bash-isms anywhere?

Reply:
Yes, Cygwin seems to use bash, at least on @fournet's and my machines, possibly everywhere. The GNU Make Manual also says that /bin/sh should be the default, so I fixed it here.

Reply:
Turns out Cygwin installs bash into `/bin/sh.exe`...

Reply:
ok; in that case, let's take a dependency on SHELL=/bin/bash, that way, everyone gets a consistent setup; we've been bitten by this in F\* already and people tend to naturally write bash-isms in Makefiles (such as: `rm -rf extract/Kremlin-{Library,Internal-Test}`